### ISSUE TITLE
Bug - CSV import using CURL needs correction on passing import parameters and schema

### ISSUE THREAD

**Issue by bsmth** [questdb/questdb.io#339]
__Bug description:__
Existing examples show how to use curl using the `-F` flag for setting parameters such as table name and designated timestamp:

```shell
curl \
  -F schema='[{"name":"userId", "type": "STRING"},{"name":"movieId", "type":"STRING"}]' \
  -F data=@ratings.csv \
  -F "overwrite=true" \
  http://localhost:9000/imp
```

The parameters currently work by appending them to the URL i.e.:

 ```bash
curl -F data=@data.csv 'http://localhost:9000/imp?name=newtable&timestamp=ts_column&partitionBy=MONTH'
```

We also require a working example of passing a schema using CURL as it seems this is ignored in the example request

__Curl version:__

```bash
curl 7.64.1 (x86_64-apple-darwin20.0) libcurl/7.64.1 (SecureTransport) LibreSSL/2.8.3 zlib/1.2.11 nghttp2/1.41.0
Release-Date: 2019-03-27
```

__Location:__
https://questdb.io/docs/reference/api/rest#user-defined-schema

**Comment by TimBo93** [comment]
`-G --data-urlencode "overwrite=true"`

This is the current behaviour in the front-end:
![image](https://user-images.githubusercontent.com/8701341/106522940-b4944580-64e0-11eb-90f2-324e4b08910e.png)

It would be good to make it clearer which parameter type works for which parameters.

**Comment by bsmth** [comment]
fixed in #382
